src/panels/config/zone/ha-config-zone.ts

```diff
@@ -186,15 +186,9 @@ export class HaConfigZone extends SubscribeMixin(LitElement) {
                       <ha-icon-button
                         .entityId=${stateObject.entity_id}
                         @click=${this._openCoreConfig}
-                        disabled=${ifDefined(
-                          stateObject.entity_id === "zone.home" &&
-                            this.narrow &&
-                            this._canEditCore
-                            ? undefined
-                            : true
-                        )}
+                        .disabled=${stateObject.entity_id === "zone.home" &&
```

Comment:
Disabling here is annoying, the user doesn't know why they can't. It's better to have it always allow editing, and tapping will show an error.